### ISSUE TITLE
The README.md is confusing

### ISSUE THREAD

**Issue by taowang1993** [frappe/lms#1073]
For self hosting, it asks me to install Bench referring to https://github.com/frappe/bench

Then, this bench repo asks me clone https://github.com/frappe/frappe_docker.git

But the pwd.yaml file contains the ERPNext application which is not what I want.

https://github.com/frappe/frappe_docker/blob/main/pwd.yml

I am confused.

**Comment by taowang1993** [comment]
I deployed frappe_docker. What should I do from here?

<img width="976" alt="image" src="https://github.com/user-attachments/assets/82c969bd-b57d-42c7-9552-c0e4e1a70302">

**Comment by taowang1993** [comment]
I gave up the deployment approach, since it is asking me install the frappe_docker. I don't know why I need that for LMS.

Instead, I just cloned this LMS repo and run docker compose up -d.

But now, I wonder how to change the default admin password?

Is there any detailed docs about this software at all?

<img width="806" alt="image" src="https://github.com/user-attachments/assets/86135ef8-d46e-4699-9b30-3e20edc4755d">
